salt/utils/data.py

```diff
@@ -86,15 +85,14 @@ def decode_dict(data):
     return rv
 
 
-@jinja_filter('json_decode_list')
 def decode_list(data):
```

Comment:
dito

Reply:
What?

Reply:
i.e. the decode instead of more English name.

Reply:
There is no reason to change it. This function is not user-facing, and its "encode" counterpart is only user-facing as a jinja filter, where the use has been clearly documented.